test262/index.html

```diff
@@ -0,0 +1,105 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Boa EcmaScript conformance</title>
+    <meta charset="UTF-8" />
+    <link rel="icon" type="image/png" href="../assets/favicon.png" />
+    <link href="../assets/bootstrap.min.css" rel="stylesheet" />
```

Comment:
These should link to the node_modules bootstrap, we shouldn’t be committing bootstrap files directly into he repo

Reply:
Same with jquery, you may need webpack to link these dependencies together

Reply:
Hmmm we are already loading them for the benchmarks, and we don't use webpack for this. Maybe we should refactor everything in that regard.

Reply:
We decided to fix this up at a later date